common/org.testeditor.dsl.common.ui/src/org/testeditor/dsl/common/ui/wizards/TestProjectConfigurationWizardPage.xtend

```diff
@@ -47,42 +50,44 @@ class TestProjectConfigurationWizardPage extends WizardPage {
 	}
 
 	override void createControl(Composite superParent) {
-		val parent = new Composite(superParent, SWT.NONE)
-		parent.layout = new GridLayout(3, false)
+		val parent = new Composite(superParent, SWT.NONE) => [layout = new GridLayout(3, false)]
 		createBuildSystemSelectionArea(parent)
 		createFixtureSelectionArea(parent)
 		createDemoSelectionArea(parent)
 		setControl(buildSystem);
 	}
 
 	private def createDemoSelectionArea(Composite parent) {
-		demoCode = new Button(parent, SWT.CHECK)
-		demoCode.text = "Generate with examples"
+		demoCode = new Button(parent, SWT.CHECK) => [text = "Generate with examples"]
```

Comment:
I think it's more readable if you put the closure into a new line, e.g.

```
demoCode = new Button(parent, SWT.CHECK) => [
    text = "Generate with examples"
]
```

And then for just a single line in method that is for creating this element only this is probably not necessary.